antenna/ext/gcs/crashstorage.py

```diff
@@ -170,3 +185,27 @@ def save_crash(self, crash_report):
 
         self.save_dumps(crash_id, dumps)
         self.save_raw_crash(crash_id, raw_crash)
+
+    def load_crash(self, crash_id):
+        """Load crash data."""
+        raw_crash = {}
+        dumps = {}
+
+        raw_crash_key = self._get_raw_crash_path(crash_id)
+        try:
+            raw_crash = json.loads(self._load_file(raw_crash_key))
+        except NotFound as exc:
+            raise CrashIDNotFound(f"{crash_id} not found") from exc
+
+        dump_names_path = self._get_dump_names_path(crash_id)
+        dump_names = json.loads(self._load_file(dump_names_path))
```

Comment:
I was reviewing the past comment history, and [this comment](https://github.com/mozilla-services/antenna/pull/995#discussion_r1561749846) suggests this check for `dump_names` would be in a `try..except` block and raise an exception if no `dump_names`.

It does look like things are consistent now, with no logging or exception raised if `dump_names` is empty. I know you said it would be very unusual for this to happen. Not sure if there's any benefit to raising an exception or anything we could do on the crash ingestion side if it were to happen (other than making sure we're pulling out the files correctly).